.evergreen/legacy_config_generator/evergreen_config_lib/tasks.py

```diff
@@ -290,7 +290,7 @@ def __init__(
     LinkTask("link-with-cmake-windows", suffix_commands=[func("link sample program MSVC")]),
     LinkTask(
         "link-with-cmake-windows-ssl",
-        suffix_commands=[func("link sample program MSVC", ENABLE_SSL="AUTO")],
+        suffix_commands=[func("link sample program MSVC", ENABLE_SSL="1")],
```

Comment:
Good catch. Recommend updating other instances of `ENABLE_SSL="AUTO"` in `LinkTask` definitions to use `"1"` for consistency.